yarn.lock

```diff
@@ -23710,6 +23711,14 @@ whatwg-url@^7.0.0:
     tr46 "^1.0.1"
     webidl-conversions "^4.0.2"
 
+wheel-gestures@^2.2.5:
+  version "2.2.47"
+  resolved "https://registry.yarnpkg.com/wheel-gestures/-/wheel-gestures-2.2.47.tgz#0324e511451b5ce643efff6c4e03f2669a058cdf"
+  integrity sha512-uE73zRoikB0Uwc4mAn6vH/X2w/E7kGe9exnEjaA3poaH+Zw2Tr9V39uMMQX3hd+wlW656dqIWBVWKY/UlP0MgQ==
+  dependencies:
```

Comment:
please report issue to this [package repo](https://github.com/xiel/wheel-gestures/issues) as it ships invalid dependencies. Those 2 deps should be specified as devDependencies, they have no use in production code used from this package - this creeps our node_modules by adding no benefits rather the opposite. ty

Reply:
Opened PR & issue here: https://github.com/xiel/wheel-gestures/pull/720

Will wait for merge before landing